### PR TITLE
add a discriiption

### PR DESCRIPTION
**Pull request by MurphySongAI**
rewrite a test.py and add new line in dockerfile

### CODE REVIEW

File: examples/paul_graham_essay/test.py [hunk 1]
```diff
@@ -7,7 +7,9 @@
...
 # query_engine.query("what is AI?")
 # index.storage_context.persist()
-response = query_engine.query("What did the author do growing up?")
-print(response)
+while True:
+    question_prompt = input("Your question: ")
+    response = query_engine.query(question_prompt)
+    print("Answer: ", response)
 #
 #
```
Comment (Quartzing) [inline examples/paul_graham_essay/test.py:14]: remove

Comment (MurphySongAI) [reply examples/paul_graham_essay/test.py:14]: why remove

Comment (Quartzing) [reply examples/paul_graham_essay/test.py:14]: it is useless

Comment (MurphySongAI) [reply examples/paul_graham_essay/test.py:14]: DONE!!!!!!!!!!!!!!!!!

Comment (Quartzing) [reply examples/paul_graham_essay/test.py:14]: thx

**Review by Quartzing — APPROVED** [review]

**Pull request merged**